makePDF.py

```diff
@@ -672,6 +714,124 @@ def do_finish(self, can_switch_to_pdf):
 			if get_setting('open_pdf_on_build', True):
 				self.view.run_command("jump_to_pdf", {"from_keybinding": False})
 
+	if _HAS_PHANTOMS:
+		def _find_errors(self, errors, error_class):
+			for line in errors:
+				m = self.file_regex.search(line)
+				if not m:
+					continue
+				groups = m.groups()
+				if len(groups) == 4:
+					file, line, column, text = groups
+				else:
+					continue
+				if line is None:
+					continue
+				line = int(line)
+				column = int(column) if column else 0
+				if file not in self.errs_by_file:
+					self.errs_by_file[file] = []
+				self.errs_by_file[file].append((line, column, text, error_class))
+
+		def create_errs_by_file(self):
+			file_regex = self.output_view.settings().get("result_file_regex")
+			if not file_regex:
+				return
+			self.errs_by_file = {}
+			try:
+				self.file_regex = re.compile(file_regex, re.MULTILINE)
+			except:
+				print("Cannot compile file regex.")
+				return
+			lt_settings = sublime.load_settings("LaTeXTools.sublime-settings")
+			level_name = lt_settings.get("show_error_phantoms")

```

Comment:
Shouldn't this just use our `get_settings()` method or is there are need to by-pass it's processing?